tests/unit/test_metrics.py

```diff
@@ -0,0 +1,46 @@
+"""Testing functions of metrics module."""
+
+from datetime import datetime
+
+import numpy as np
+import polars as pl
+import pytest
+
+from wristpy.core.models import Measurement
+from wristpy.processing.metrics import euclidean_norm_min_one
+
+TEST_LENGTH = 100
+
+
+@pytest.mark.parametrize(
+    "x,y,z, expected_enmo",
+    [
+        (np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 0),
+        (np.ones(TEST_LENGTH), np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 0),
+        (np.ones(TEST_LENGTH) * 2, np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 1),
+        (np.ones(TEST_LENGTH), np.ones(TEST_LENGTH) * 2, np.ones(TEST_LENGTH) * 2, 2),
+    ],
+)
+def test_euclidean_norm_min_one(
+    x: np.ndarray, y: np.ndarray, z: np.ndarray, expected_enmo: float
+) -> None:
+    """Tests the euclidean norm function."""
+    dummy_date = datetime(2024, 5, 2)
+    dummy_datetime = pl.Series("time", [dummy_date] * TEST_LENGTH)
+    test_acceleration = Measurement(
```

Comment:
in theory you can use my readers.unix_epoch_time_to_polars_datetime function here, probably not necessary

Reply:
I prefer this actually - reduces the number of dependencies.